### PR TITLE
Feature-be/game into develop-be

### PR DESCRIPTION
**Pull request by bkkmw**
### TDD
- I tried... But still got a lot of works to do.
- Please feel free to share idea about TEST codes, TDD methodology, ...
- All the test codes are about `Unit-Test`, No `Integration Test` exists yet. #29 
- In Controller Test, I did NOT performed test about `validation check`. Think I need more time to study more about it #34 

### Transaction managing
- as others ain't not finished. Couldn't check transactions are working properly #32 #31 
- Will add some test codes to check transaction managing. ASAP
- In short, better think that NOTHING WORKS properly yet

### User Unique Identifier
- For both signed up, did not signed up : User identifier is NOT working at all
- Both will be fixed as User/Identifier is determined #27

### CODE REVIEW

**Comment by bkkmw** [comment]
Closes #29

**Comment by JODONG2** [comment]
친절한 설명 감삼다. 덕분에 이해가 잘됐어요. 
Comment랑 합쳐보겠습니당. 감삼다 !
